### PR TITLE
[W4.11e][F11-1]Daniel Ng

### PR DESCRIPTION
**Pull request by ghost**
LO W4.11e

### CODE REVIEW

**Comment by ghost** [comment]
Ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,41 @@ private void assertAreUnique(Object... objects) {
...
+
+        // there are no null objects
+        assertHasNoNull("abc", "ab", "a");
+        assertHasNoNull(1, 2);
+        assertHasNoNull(objectList);
+        assertHasNoNull(objectList, "def", "ghi", "jkl");
+
+        // some null objects
+        assertHasNoNull("abc", "abc");
+        assertHasNoNull("abc", "", "abc", "ABC");
```
Comment (liliwei25) [inline test/java/seedu/addressbook/common/UtilsTest.java]: Line 67 and line 68 is repetitive. I think just one will do.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,41 @@ private void assertAreUnique(Object... objects) {
...
+        assertHasNoNull("", "abc", "a", "abc");
+        assertHasNoNull(1, Integer.valueOf(1));
+        assertHasNull(null, 1, Integer.valueOf(1));
+        assertHasNull(null, null);
```
Comment (liliwei25) [inline test/java/seedu/addressbook/common/UtilsTest.java]: I think it'll be neater and more readable for others if you arrange the tests according to the number of objects

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -43,4 +43,41 @@ private void assertAreUnique(Object... objects) {
...
+        assertHasNoNull(1, 2);
+        assertHasNoNull(objectList);
+        assertHasNoNull(objectList, "def", "ghi", "jkl");
+
+        // some null objects
+        assertHasNoNull("abc", "abc");
+        assertHasNoNull("abc", "", "abc", "ABC");
+        assertHasNoNull("", "abc", "a", "abc");
+        assertHasNoNull(1, Integer.valueOf(1));
+        assertHasNull(null, 1, Integer.valueOf(1));
```
Comment (liliwei25) [inline test/java/seedu/addressbook/common/UtilsTest.java]: It'll be more readable to add a blank line between the assertHasNoNull and assertHasNull checks

**Review by liliwei25 — COMMENTED** [review]
Good job! 👍 

Please close the PR after reading the comments!

**Pull request closed without merging**